src/modules/device.json

```diff
@@ -244,9 +244,17 @@
               "$ref": "https://meta.comcast.com/firebolt/types#/definitions/SemanticVersion",
               "description": "The Firebolt SDK version"
             },
+            "api": {
+              "$ref": "https://meta.comcast.com/firebolt/types#/definitions/SemanticVersion",
+              "description": "The lateset Firebolt API vresion supported by the curent device."
+            },
+            "firmware": {
+              "$ref": "https://meta.comcast.com/firebolt/types#/definitions/SemanticVersion",
+              "description": "The device firmware version."
+            },
             "os": {
               "$ref": "https://meta.comcast.com/firebolt/types#/definitions/SemanticVersion",
-              "description": "The Firebolt OS version"
+              "description": "**Deprecated**"
```

Comment:
Usually, deprecating something means that you keep the thing around for a minute Until everyone has migrated. It would also be nice if the `description` field here could provide a suggestion for what to use instead.

Reply:
fixed